raiden-ts/src/channels/utils.ts

```diff
@@ -87,11 +92,15 @@ export function channelAmounts(channel: Channel) {
     ownPendingWithdraw = bnMax(
       // get maximum between actual and pending withdraws (as it's a total)
       ownWithdraw,
-      ...channel.own.withdrawRequests.map((req) => req.total_withdraw),
+      ...channel.own.pendingWithdraws
+        .filter(Signed(WithdrawRequest).is)
+        .map((req) => req.total_withdraw),
```

Comment:
It's code style. In the old days, it was 80 chars (and this is still the limit in the dApp), on SDK we use 99chars as limit. This is just to avoid long long lines, which usually don't fit well in the screen. It was more of a matter when we had 44x80 green-phosphorus screen, but to this day is still considered a good practice and adopted by us.